2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
```

Comment:
Oh, I'm specifically talking about managing "state of tessel" updates / announcements so that when people ask me about the state of tessel, I can point them to the most recent source of truth, which should probably be the first thing you see on the website when you hit tessel.io.